azurerm/internal/services/cognitive/tests/resource_arm_cognitive_account_test.go

```diff
@@ -202,6 +225,24 @@ resource "azurerm_resource_group" "test" {
   location = "%s"
 }
 
+resource "azurerm_cognitive_account" "test" {
+  name                = "acctestcogacc-%d"
```

Comment:
Would we not change `"acctestcogacc-%d"` to `"acctestCOGacc-%d" or `"acctestCOGACC-%d"`?

Reply:
Thats something to tackle in a different PR